docs/skopeo.1.md

```diff
@@ -12,26 +12,31 @@ It also allows you to copy container images between various registries, possibly
 ## IMAGE NAMES
 Most commands refer to container images, using a _transport_`:`_details_ format. The following formats are supported:
 
-  **atomic:**_namespace_**/**_stream_**:**_tag_
-  An image in the current project of the current default Atomic
-  Registry. The current project and Atomic Registry instance are by
-  default read from `$HOME/.kube/config`, which is set e.g. using
-  `(oc login)`.
+  **atomic:@**_hostname_**/**_namespace_**/**_stream_**:**_tag_
+  An image served by an OpenShift(Atomic) Registry server. The current OpenShift project and OpenShift Registry instance are by default read from `$HOME/.kube/config`, which is set e.g. using `(oc login)`.
+
+  **container-storage://**_docker-reference_
+  An image in local container/storage format.  Location and image store specified in /etc/containers/storage.conf
 
   **dir:**_path_
-  An existing local directory _path_ storing the manifest, layer
-  tarballs and signatures as individual files. This is a
-  non-standardized format, primarily useful for debugging or
-  noninvasive container inspection.
+  An existing local directory _path_ storing the manifest, layer tarballs and signatures as individual files. This is a non-standardized format, primarily useful for debugging or noninvasive container inspection.
 
   **docker://**_docker-reference_
-  An image in a registry implementing the "Docker Registry HTTP API V2".
-  By default, uses the authorization state in `$HOME/.docker/config.json`,
-  which is set e.g. using `(docker login)`.
+  An image in a registry implementing the "Docker Registry HTTP API V2". By default, uses the authorization state in `$HOME/.docker/config.json`, which is set e.g. using `(docker login)`.
+  
+  **docker-archive:**path[**_docker-reference_]
+  docker-archive indicates the image as stored in the `docker save` formated file.  docker-regerence is only used when creating such file, and it must not contain a digest.
+
+  **docker-daemon:**_docker-reference_
+  An image _docker-reference_ stored in the docker daemon internal storage.  _docker-reference_ must container a tag or a digest.  Alternatively, when reading images, the format can also be docker-daemon:algo:digest (an image ID).
 
   **oci:**_path_**:**_tag_
-  An image _tag_ in a directory compliant with "Open Container Image
-  Layout Specification" at _path_.
+  An image _tag_ in a directory compliant with "Open Container Image Layout Specification" at _path_.
+
+  **ostree:**_path_
+  An image in local OSTree repository.
+
+  The format is **ostree:**_image_ or **ostree:**_image_**@/absolute/repo/path, with /absolute/repo/path defaulting to /ostree/repo. Image is not quite a dockerlike-reference (e.g. no docker.io/library normalization is used), but it either should contain a :tag, or an implied :latest is used.
```

Comment:
The ostree-related descriptions should be a single paragraph. Misses terminating `**` around `@` (`**@**`), so the initial ones show up in the output. `/ostree/repo` should be marked as a filename, probably using bold (`**` again).

Perhaps drop the `is not quite … but it either` part? It is too much detail about what the semantics _isn’t_, and we would have two fewer references to… that word :)